supervisord.conf

```diff
@@ -0,0 +1,42 @@
+[supervisord]
+
+[unix_http_server]
+file=/var/run/supervisor.sock
+chmod=0700
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl=unix:///var/run/supervisor.sock
+
+[program:api]
+command=python3 run_processor_api.py --config %(ENV_CONFIG_FILE)s
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
```

Comment:
maxbytes = 0 means unlimited? What if we put a limit here, say 200Mb for each log? If this is going to be running possibly for years then that would be good

Reply:
The `stdout_logfile_maxbytes` parameter defines the number of bytes required to rotate the log file (http://supervisord.org/logging.html#activity-log-rotation). The value 0 means `the logfile is never rotated `. 
We are using `/dev/stdout` as logfile, so the rotation is not required.